src/main/scala/nl/knaw/dans/bag/Deposit.scala

```diff
@@ -367,6 +384,21 @@ object Deposit {
       } yield new Deposit(baseDir, bag, properties)
   }
 
+  //TODO wouldn't it be better to be able to provide the payloadDir and a baseDir for the Deposit separately?
```

Comment:
Or as a separate method. I guess it could be used by `easy-deposit-api` to copy the deposit from the draft-area to the staging-area

Reply:
no, a deposit is created in the draft-area and then moved/copied to the staging area. I don't see the need for a dans-bag-lib method for that. Just use `better.files.File.move/copy`

What I meant to question with this is: '_Would it make sense to have an extra method that doesn't create the bag in-place, but rather at a different location, indicated by `baseDir`?_'. However, after thinking somewhat more about this, the user of this library can just do that by itself by first copying the original data to `baseDir` and then calling this method.

If you agree, feel free to remove this line.

Reply:
@jo-pol and i thought to use such a method to automatically check the result of the copy by validating the checksums